internal/cli/atlas/privateendpoints/gcp/interfaces/delete.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interfaces
+
+import (
+	"context"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/cli/require"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DeleteOpts struct {
+	cli.GlobalOpts
+	*cli.DeleteOpts
+	privateEndpointServiceID string
+	privateEndpointGroupID   string
+	store                    store.InterfaceEndpointDeleter
+}
+
+func (opts *DeleteOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+func (opts *DeleteOpts) Run() error {
+	return opts.Delete(opts.store.DeleteInterfaceEndpoint, opts.ConfigProjectID(), provider, opts.privateEndpointServiceID, opts.privateEndpointGroupID)
+}
+
+// mongocli atlas privateEndpoint(s) gcp interface(s) delete <endpointGroupId> --endpointServiceId endpointServiceId [--projectId projectId].
+func DeleteBuilder() *cobra.Command {
+	opts := &DeleteOpts{
+		DeleteOpts: cli.NewDeleteOpts("Interface endpoint '%s' deleted\n", "Interface endpoint not deleted"),
+	}
+	cmd := &cobra.Command{
+		Use:     "delete <endpointGroupId>",
+		Aliases: []string{"rm"},
+		Short:   "Delete a specific GCP private endpoint interface for your project.",
+		Args:    require.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(opts.ValidateProjectID, opts.initStore(cmd.Context()))
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			opts.Entry = args[0]
+			if err := opts.Prompt(); err != nil {
+				return err
+			}
+			return opts.Run()
+		},
+	}
+
+	cmd.Flags().BoolVar(&opts.Confirm, flag.Force, false, usage.Force)
+	cmd.Flags().StringVar(&opts.privateEndpointServiceID, flag.EndpointServiceID, "", usage.EndpointServiceID)
+
```

Comment:
Redundant space?

Reply:
This is a straight copy from the equivalent AWS delete code, which in turn is identical to the Azure delete code.

Reply:
But I agree: no need for redundant whitespace, so I have removed it in the latest commit.